src/components/ProgramRecordSendModal/SendLearnerRecordModal.scss

```diff
@@ -1,17 +1,20 @@
 .send-record-modal {
-    @extend .d-flex, .flex-column;
+    display: flex !important;
+    flex-direction: column !important;
```

Comment:
Based on https://github.com/openedx/frontend-app-authn/pull/1436#discussion_r2079869474 moving away from `@extend` in this way is good as-is!